### PR TITLE
PMM-6671 add HAProxy add instance form

### PR DESCRIPTION
**Pull request by lunaticusgreen**

### CODE REVIEW

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      <div className={styles.labelWrapper} data-qa="address-label">
+        <span>{Messages.form.labels.mainDetails.address}</span>
+        <LinkTooltip tooltipText={Messages.form.tooltips.mainDetails.address} icon="info-circle" />
+      </div>
+      <TextInputField
+        name="address"
+        placeholder={Messages.form.placeholders.mainDetails.address}
+        validators={[validators.required]}
+      />
+      <div className={styles.labelWrapper} data-qa="service_name-label">
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx:29]:
```suggestion
      <div className={styles.labelWrapper} data-qa="service-name-label">
```

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <span>{Messages.form.labels.mainDetails.serviceName}</span>
+        <LinkTooltip tooltipText={Messages.form.tooltips.mainDetails.serviceName} icon="info-circle" />
+      </div>
+      <TextInputField name="service_name" placeholder={Messages.form.placeholders.mainDetails.serviceName} />
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx:33]:
```suggestion
      <TextInputField name="service-name" placeholder={Messages.form.placeholders.mainDetails.serviceName} />
```

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx (hunk 1 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx:29]: done

File: pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx (hunk 2 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.tsx:33]: done

**Review by tiagomotasantos — DISMISSED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**